src/resources/js/tickets-block.js

```diff
@@ -241,5 +241,224 @@ tribe.tickets.block = {
 
 	obj.init();
 
+	/*
+	 * AR Cart Modal Selectors
+	 *
+	 * @since TBD
+	 *
+	 */
+	obj.modalSelector = {
+		container: '.tribe-modal__cart',
+		item: '.tribe-block__tickets__item',
+		itemQuantity: '.tribe-ticket-quantity',
+		itemPrice: '.tribe-block__tickets__item__extra__price',
+		itemTotal: '.tribe-modal__cart__item__total',
+		itemRemove: '.tribe-modal__cart__item__remove',
+		cartTotals: '.tribe-modal__cart__totals',
+		cartQuantity: '.tribe-modal__cart__total__qty',
+		cartTotal: '.tribe-modal__cart__total__amount',
+	};
+
+	/**
+	 * On Change of Modal Cart Qty Update Item
+	 *
+	 * @since TBD
+	 *
+	 */
+	$( document ).on( 'change', obj.modalSelector.container + ' ' + obj.modalSelector.itemQuantity, function ( e ) {
+			e.preventDefault();
+
+			var $cartItem = $( this ).closest( obj.modalSelector.item );
+			var id = $cartItem.data( 'ticketId' );
+			var ticket = obj.updateItem( id, $cartItem );
+
+			$( te ).trigger( 'tribe_tickets_qty_updated', ticket );
+
+		}
+	);
+
+	/**
+	 * Remove Item from Cart Modal
+	 *
+	 * @since TBD
+	 *
+	 */
+	$( document ).on( 'click', obj.modalSelector.itemRemove, function ( e ) {
+			e.preventDefault();
+
+			var $cartItem = $( this ).closest( obj.modalSelector.item );
+			$cartItem.find( obj.modalSelector.itemQuantity ).val( 0 );
+			$cartItem.fadeOut() ;
+
+			var ticket = {};
+			ticket.id = $cartItem.data( 'ticketId' );
+			ticket.qty = 0;
+			ticket.price = obj.getPrice( $cartItem, obj.modalSelector.itemPrice );
+
+			obj.updateTotal( ticket.qty, ticket.price, $cartItem );
+
+			$( te ).trigger( 'tribe_tickets_qty_updated', ticket );
+		}
+	);
+
+	/**
+	 * When Modal Qty is updated, update the cart totals in modal
+	 *
+	 * @since TBD
+	 *
+	 */
+	$( te ).on( 'tribe_tickets_qty_updated', function ( e, ticket ) {
+
+		var $cart = $( obj.modalSelector.container );
+
+		obj.updateCartTotals( $cart );
+
+	} );
+
+	/**
+	 * When Get Tickets CLicked Update the Modal
+	 *
+	 * @since TBD
+	 *
+	 */
+	$( te ).on( 'tribe_tickets_get_tickets', function ( e, dialogEl, event ) {
+
```

Comment:
This uses the trigger added here: https://github.com/moderntribe/tribe-common/pull/1125

Reply:
add a new custom trigger that can be changed per modal/dialog in the above common PR that has been merged.